docs/docs/releases/0.2.1.md

```diff
@@ -0,0 +1,152 @@
+# Mathesar 0.2.1
+
+## Summary
+
+Mathesar 0.2.1 addresses a number of bug fixes identified by members of our community during our **beta** release. This release also includes some quality-of-life improvements to Mathesar like nicknames for databases, persisted column widths, and the ability to connect databases without a password.
+
+We also have a new form to allow users to submit suggestions and feedback about Mathesar to our team.
+
+_This page provides a comprehensive list of all changes since version [0.2.0](./0.2.0.md)._
+
+<!-- ## Breaking changes -->
+<!-- (This section lists any breaking changes to publicly exposed and documented machine interfaces to Mathesar such as the API or DB functions) -->
+
+
+## Improvements
+<!-- (Each feature within this section should have its own level-three heading) -->
+
+### Database nicknames
+
+[#4262](https://github.com/mathesar-foundation/mathesar/pull/4262 "Frontend support for databases nicknames")
+[#4244](https://github.com/mathesar-foundation/mathesar/pull/4244 "Backend support for database nicknames")
+[#4261](https://github.com/mathesar-foundation/mathesar/pull/4261 "Allow DB name/host/port to be updated")
+
+### In-app feedback
+
+[#4241](https://github.com/mathesar-foundation/mathesar/pull/4241 "Move feedback to app header, add mailing list link to home page")
+
+### Column widths are persisted
+
+[#4257](https://github.com/mathesar-foundation/mathesar/pull/4257 "Persist column width")
+[#4248](https://github.com/mathesar-foundation/mathesar/pull/4248 "Backend support to persist customized column widths")
+
+### Docker Compose installs are updated to Postgres 17
+
+!!! info "For existing users"
+
+    Existing users who would like to update to Postgres 17 should use our new docker compose file and run `cp -r msar msar.backup` prior to upgrading Postgres in the event of issues.
```

Comment:
This needs more explanation. It's not necessarily true that a user will know what's referred to by `msar`. This should be steps. Something like:

If you'd like to upgrade your Mathesar-managed Postgres server to the newest version,

1. Navigate to your installation directory.
2. `docker compose down`
3. `cp -r msar msar.backup`
4. `cp docker-compose.yml docker-compose.yml.backup`
5. `wget https://blah.blah/docker-compose.yml` 
6. Edit the new docker compose file with your variables.
7. `docker compose up --pull always -d`
8. Remove backups once you're satisfied.

These steps could be here, or in the upgrade section below, but they need to be somewhere.

Reply:
Added in https://github.com/mathesar-foundation/mathesar/commit/c209b5222a4aa605b3a5086d3c1fceb33512fbb2#diff-773fdf4c10d695aaf2fc639f089d53386f9927dd71c33866fb3d42195ee8bc6bR97